include/envoy/network/io_handle.h

```diff
@@ -57,6 +57,27 @@ class IoHandle {
    * err_ = nullptr and rc_ = the bytes written for success.
    */
   virtual Api::IoCallUint64Result writev(const Buffer::RawSlice* slices, uint64_t num_slice) PURE;
+
+  /**
+   * Send buffer to address.
+   * @param buffer points to the location of the data to be sent.
+   * @param size indicates the size of the data in bytes.
+   * @param to_address is the destination address.
+   * @return a Api::IoCallUint64Result with err_ = an Api::IoError instance or
+   * err_ = nullptr and rc_ = the bytes written for success.
+   */
+  virtual Api::IoCallUint64Result sendto(const void* buffer, size_t size, int flags,
```

Comment:
Can we just pass a single `Buffer::RawSlice` here instead of a buffer and size?

Reply:
Sure